main.tf

```diff
@@ -23,14 +23,10 @@ locals {
   gcloud_bin_abs_path  = abspath(local.gcloud_bin_path)
   components           = join(" ", var.additional_components)
 
-  gcloud  = "${local.gcloud_bin_path}/gcloud"
-  gsutil  = "${local.gcloud_bin_path}/gsutil"
-  bq      = "${local.gcloud_bin_path}/bq"
-  kubectl = "${local.gcloud_bin_path}/kubectl"
-  jq      = "${local.gcloud_bin_path}/jq"
+  gcloud = var.skip_download ? "gcloud" : "${local.gcloud_bin_path}/gcloud"
```

Comment:
This is assuming gcloud is in the process's PATH. Could we make `gcloud_bin_path` a variable as well that if left default will use the local? Then concat that with `create_cmd_entrypoint`? Or would this break some use cases you know of?

Reply:
We *could* but I'd probably make that a separate change.

In particular, for shared modules we don't necessarily *know* the user's gcloud location. Just that they want to use whatever `gcloud` is in their path.

Reply:
Seems fair